tests/ConfigurationValidatorTest.php

```diff
@@ -1,87 +1,73 @@
-            try {
-                ConfigurationValidator::validate([
-                    'feed1' => [
-                        'view' => 'feed::rss',
-                        'format' => $format,
-                    ],
-                ]);
-            } catch (InvalidConfiguration $e) {
-                $exceptionCounter++;
-            }
-        }
+test('it validates feed formats', function () {
+    $exceptionCounter = 0;
+    $formats = ['atom', 'json', 'rss', 'other'];
 
-        $this->assertEquals(1, $exceptionCounter);
+    foreach ($formats as $format) {
+        try {
+            ConfigurationValidator::validate([
+                'feed1' => [
+                    'view' => 'feed::rss',
+                    'format' => $format,
+                ],
+            ]);
+        } catch (InvalidConfiguration $e) {
+            $exceptionCounter++;
+        }
     }
+    assertEquals(1, $exceptionCounter);
+});
 
-    /** @test */
-    public function it_throws_an_exception_for_invalid_feed_types()
-    {
-        $this->expectException(InvalidConfiguration::class);
-
-        ConfigurationValidator::validate([
-            'feed1' => [
-                'view' => 'feed::rss',
-                'format' => 'test',
-            ],
-        ]);
-    }
+test('it throws an exception for invalid feed types', function () {
+    ConfigurationValidator::validate([
+        'feed1' => [
+            'view' => 'feed::rss',
+            'format' => 'test',
+        ],
+    ]);
+})->throws(InvalidConfiguration::class);
 
-    /** @test */
-    public function it_throws_an_exception_for_an_invalid_items_value()
-    {
-        $exceptionCounter = 0;
+test('it throws an exception for an invalid items value', function () {
+    $exceptionCounter = 0;
 
-        $invalidItems = [[], null, '', ['test']];
-        $validItems = ['Model@getAll', ['App\\Model', 'getItems'], ['App\\Model', 'getItems', 'param1']];
+    $invalidItems = [[], null, '', ['test']];
+    $validItems = ['Model@getAll', ['App\\Model', 'getItems'], ['App\\Model', 'getItems', 'param1']];
 
-        $items = array_merge($invalidItems, $validItems);
+    $items = array_merge($invalidItems, $validItems);
 
-        foreach ($items as $itemsValue) {
-            try {
-                ConfigurationValidator::validateResolver('feed1', $itemsValue);
-            } catch (InvalidConfiguration $e) {
-                $exceptionCounter++;
-            }
+    foreach ($items as $itemsValue) {
+        try {
+            ConfigurationValidator::validateResolver('feed1', $itemsValue);
+        } catch (InvalidConfiguration $e) {
+            $exceptionCounter++;
         }
-
-        $this->assertEquals(count($invalidItems), $exceptionCounter);
     }
 
-    /** @test */
-    public function it_throws_an_exception_for_an_invalid_view()
-    {
-        $exceptionCounter = 0;
-        $views = ['', 'feed::missing', null, 'feed::rss'];
+    assertEquals(count($invalidItems), $exceptionCounter);
```

Comment:
Instead of the PHPUnit assertion, use Pest's expectation API here and everywhere assertions are made.

Reply:
Sure.